mitodl/mitxonline
# 1573 sync extended profile data to hubspot

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/1573

#### What's this PR do?
Adds all of the user profile fields from MITx Online to Hubspot.

#### How should this be manually tested?

1. You'll need some new .env settings:

```
MITOL_HUBSPOT_API_PRIVATE_TOKEN=<ask_me, and I can also add you to the hubspot account>
MITOL_HUBSPOT_API_ID_PREFIX=<your_initials>-mitxonline-dev
HUBSPOT_PIPELINE_ID=19817792
```
2. Register a user through your local MITx Online instance
3. View the Hubspot contact record created for the user in step 2 (https://app.hubspot.com/contacts/23586992/objects/0-1/views/all/list?redirectFrom=crm-records-ui) and verify that the contact record contains properties of all the populated MITx Online user profile attributes.
4. Make an update to the user's MITx Online profile and verify that the Hubspot contact properties reflect the update.
